akka-serialization-jackson/src/main/resources/reference.conf

```diff
@@ -24,9 +24,17 @@ akka.serialization.jackson {
 }
 #//#jackson-modules
 
+#//#algorithm
+akka.serialization.jackson {
+  # gzip - using common java gzip
+  # lz4 - LZ4 compression.
+  algorithm = gzip
```

Comment:
more specifc: `compression-algorithm`

but it's better if we add a subsection for `compression`

```
akka.serialization.jackson.compression {
  algorithm = gzip
  compress-larger-than = 32 KiB
}
```

Reply:
Updated to use `akka.serialization.jackson.compression`.